### ISSUE TITLE
[Feature] Bigger Help PopUp

### ISSUE THREAD

**Issue by Disonantemus** [Beastwick18/nyaa#7]
Should be at least 20 (~ double in Normal Help) lines or just show all Key/Action pairs.
Is annoying to scroll down with so much space wasted.
~~~yaml
┌Help: Normal──────────────────────┐
│      Key             Action      │
│          Enter ⇒ Confirm         █
│              q ⇒ Exit App        █
│              g ⇒ Top             █
│              G ⇒ Bottom          │
│           j, ↓ ⇒ Down            │
│           k, ↑ ⇒ Up              │
│        n, l, → ⇒ Next Page       │
└──────────────────────────────────┘
~~~
Right now is only 10 rows including borders.

**Comment by Beastwick18** [comment]
I'll change this. I'll make it around 25-30 by default, but capped at the size of the users terminal.

**Comment by Beastwick18** [comment]
![image](https://github.com/Beastwick18/nyaa/assets/13501575/b5c6637a-1aef-4a71-9eed-a0f5f0da0f29)
Fixed with 31a7d76
